Dockerfile

```diff
@@ -1,25 +1,49 @@
 FROM python:3.9-slim-bullseye as base
-ENV POETRY_PATH=/opt/poetry \
-    POETRY_VERSION=1.1.6
-ENV PATH="$POETRY_PATH/bin:$VENV_PATH/bin:$PATH"
+ENV POETRY_HOME=/etc/poetry \
+    POETRY_VERSION=1.2.1
```

Comment:
Use latest 1.3 Poetry version. Do not to rebuild and check

Reply:
Added the latest version.